CHANGELOG.md

```diff
@@ -21,6 +20,7 @@ specified in a field description cannot be found was removed.
   - `sonata.admin.label.strategy.native`
   - `sonata.admin.label.strategy.noop`
   - `sonata.admin.label.strategy.underscore`
+- Public access of `AbstractAdmin::configureActionButtons` was changed to protected
```

Comment:
Sorry I forgot this part.

Could you replace also this line by:

```markdown
`AbstractAdmin::configureActionButtons` method is now protected
```

Thanks.